internal/querycoord/task.go

```diff
@@ -320,60 +318,29 @@ func (lct *loadCollectionTask) execute(ctx context.Context) error {
 	defer cancel2()
```

Comment:
functions loadCollectionTask execute is too long, better to seperate into multiple part